tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature

```diff
@@ -25,17 +25,6 @@ Feature: files and folders can be deleted from the trashbin
     And user "Alice" has logged in using the webUI
     And the user has browsed to the trashbin page
 
-  @issue-1725 @smokeTest @ocisSmokeTest
-  Scenario: Delete files and check that they are gone
-    When the user deletes file "lorem.txt" using the webUI
-    And the user deletes file "sample,1.txt" using the webUI
-    And the user opens folder "simple-folder" using the webUI
-    Then file "lorem.txt" should not be listed on the webUI
-    And file "sample,1.txt" should not be listed on the webUI
-    But file "lorem.txt" should be listed in the trashbin folder "simple-folder" on the webUI
-    And file "lorem-big.txt" should not be listed in the trashbin folder "simple-folder" on the webUI
-    But file "lorem-big.txt" should be listed on the webUI
```

Comment:
This scenario should be just like the one below it and should work

Reply:
IMO, better remove this scenario.
Also the scenario below doesn't make sense in this feature file.
There might be similar features in `deleteFilesFolders.feature` file

Reply:
Thanx. I didn't notice that. They are in `deleteFilesFolders.feature`.

Reply:
The scenarios below are for the deletion of folders from the trash bin. I will change the scenario description.

Reply:
oh yeah, I didn't look in the background.